### PR TITLE
Make workspace output dir absolute path

### LINKED ISSUE #1118: `spago test` fails when output folder specified.
It seems like the command `test` when also specifying the `--output` flag doesn't work as expected. 

**How to reproduce**
1. Setup a dummy project
```
$ mkdir purescript-pasta
$ cd purescript-pasta
$ spago init
```
2. Build using the output folder:
```
$ spago build --output /tmp/output
```
3. Test using the compiled modules in the previous step:
```
$ spago test --output /tmp/output
```

I get this error:

```
Running tests for package: pasta
node:internal/errors:496
    ErrorCaptureStackTrace(err);
    ^

Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/cebrian/borralodedentro/purescript-pasta//tmp/output/Test.Main/index.js' imported from /home/cebrian/borralodedentro/purescript-pasta/.spago/run/run.js
    at new NodeError (node:internal/errors:405:5)
    at finalizeResolution (node:internal/modules/esm/resolve:327:11)
    at moduleResolve (node:internal/modules/esm/resolve:946:10)
    at defaultResolve (node:internal/modules/esm/resolve:1132:11)
    at nextResolve (node:internal/modules/esm/loader:163:28)
    at ESMLoader.resolve (node:internal/modules/esm/loader:835:30)
    at ESMLoader.getModuleJob (node:internal/modules/esm/loader:424:18)
    at ModuleWrap.<anonymous> (node:internal/modules/esm/module_job:77:40)
    at link (node:internal/modules/esm/module_job:76:36) {
  code: 'ERR_MODULE_NOT_FOUND'
}

Node.js v18.18.0
```

Notice that the output folder has been concatenated to the current folder instead of dealing with it as absolute in /home/cebrian/borralodedentro/purescript-pasta/ **/tmp/output/** Test.Main/index.js

### PR DESCRIPTION
**Pull request by JordanMartinez**
### Description of the change

Fixes #1118

### Checklist:

- [ ] Added the change to the "Unreleased" section of the changelog
- [ ] Added some example of the new feature to the `README`
- [ ] Added a test for the contribution (if applicable)

**P.S.**: the above checks are not compulsory to get a change merged, so you may skip them. However, taking care of them will result in less work for the maintainers and will be much appreciated 😊

### CODE REVIEW

File: src/Spago/Command/Run.purs [hunk 1]
```diff
@@ -85,9 +87,7 @@ run = do
         nodeContents =
           Array.fold
             [ "import { main } from 'file://"
-            , withForwardSlashes opts.sourceDir
-            , "/"
-            , fromMaybe "output" workspace.buildOptions.output
+            , absOutput
```
Comment (f-f) [inline src/Spago/Command/Run.purs:90]: CI for windows is failing and I suspect it's due to this
```suggestion
            , withForwardSlashes absOutput
```

**Review by f-f — COMMENTED** [review]
Do we need to treat paths as absolute throughout the program?
I think we could get away with just calling `Path.resolve` at the usage location.

**Comment by JordanMartinez** [comment]
Latest CI error was due to a test timing out in a parsing test. I don't think that's related to this PR.

**Review by f-f — APPROVED** [review]
Yes! This is great 🙂

**Pull request merged**